extensions/value_generators/templates/Copier.js

```diff
@@ -17,7 +17,7 @@ oppia.directive('copier', function($compile) {
   return {
     link: function(scope, element, attrs) {
       scope.getTemplateUrl = function() {
-        return VALUE_GENERATOR_TEMPLATES_URL + scope.$parent.generatorId;
+        return '/value_generator_handler/' + scope.$parent.generatorId;
```

Comment:
I don't think I want to do it for this one -- I was chatting with @amitdeutsch today and we're planning to get rid of the whole value generator thing altogether. So this is going to end up disappearing.